TurkishFilter/sections/specific.txt

```diff
@@ -83,6 +83,9 @@ tekkocaeli.com##.sagkulereklam
 ||tekkocaeli.com/wp-content/uploads/2018/10/377_8.jpg
 ||gercekkocaeli.com.tr/resimler/images/Teknocom(1).jpg
 gercekkocaeli.com.tr##ul[style="width:400px; height:300px;"]
+celtabettv5.com#$#body { background-image: none!important; }
+celtabettv5.com##.siteAnnouncer
+||celtabettv5.com/*/banners/*.gif
```

Comment:
use existing `/uploads/banners/*$domain=`

Reply:
added domain to existing rule

Reply:
moved #$#-rule to general_extensions